py/strato/common/log/log2text.py

```diff
@@ -101,15 +147,15 @@ def follow_generator(istream):
 
 def printLog(logFile, formatter, follow):
     inputStream = sys.stdin if logFile == "-" else open(logFile)
+    logConf = formatter._getLogTypeConf(logFile)
     if follow:
         inputStream = follow_generator(inputStream)
     for line in inputStream:
         try:
-            obj = json.loads(line)
-            formatted = formatter.process(obj)
+            formatted = formatter.process(line, logFile, logConf)
```

Comment:
doesnt the formatter.process returns 2 values now? Expect both, and ignore the second. I think explicity is better